### PR TITLE
Update policy-csp-devicehealthmonitoring.md

### PR DESCRIPTION
**Pull request by andrewjohnporter**
It would be helpful if information on what registry entries these settings create on a device and also the equivalent Group Policy admx file info.

This would help to know if the policy is applying and also if there is a conflict with Group Policy.

The registry settings I have added are the ones I believe are created by the endpoint and windows updates csp's but I can't 100% confirm this.

## Why

- Closes #[Issue Number]

### CODE REVIEW

File: windows/client-management/mdm/policy-csp-devicehealthmonitoring.md [hunk 1]
```diff
@@ -180,10 +180,14 @@ Configure this policy manually only when explicitly instructed to do so by a Mic
 <!--/Policy-->
 <hr/>
 
+**This policy creates the following two registry entres:**
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-devicehealthmonitoring.md:183]:
```suggestion
This policy creates the following two registry entries:

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @andrewjohnporter, kindly commit the suggested changes. Many thanks.

**Comment by andrewjohnporter** [comment]
All Good. Thanks.

**Comment by vinaypamnani-msft** [comment]
@andrewjohnporter We're working on automating the CSP docs, and the new format will list the registry keys associated with a CSP Policy if they have a GP Mapping defined. I'm closing this PR at this point, and will review this CSP to see if proper registry keys are listed for this CSP in the new format.

**Pull request closed without merging**